### PR TITLE
Iexfilter Patch

### PR DESCRIPTION
**Pull request by JedwardsEdwards**
v1.7.1 patch for iexfilter modifications:

- Update file loading to use inbuilt TorQ loading functionality

- Remove underscores in variable names, excluding table names to avoid clashes with historical data

- Remove obsolete fields in table schemas

- Adapt table formatting to pull from CSV config files

### CODE REVIEW

File: code/iexfeed/iex.q [hunk 1]
```diff
@@ -1,95 +1,96 @@
...
+   .iex.timer:$[not .iex.reqtype in key .iex.timerDict;'`timer;.iex.timerDict .iex.reqtype];
    };
 
-get_data:{[main_url;suffix].Q.hg`$main_url,suffix};
+getData:{[mainUrl;suffix].Q.hg`$mainUrl,suffix};
 
-format_data:{[data;cfg]
-   :update .iex.convert_epoch srctime,ex:" " from cfg[`cols] xcol enlist cfg[`types]$cfg[`scols]#data;
+formatData:{[data;cfg]
+  cfg:("ScS";enlist",")0:hsym`$cfg;
```
Comment (ThomasSmyth) [inline code/iexfeed/iex.q]: There'll be overheard from parsing the cfg file each time you do this. You can parse it when the process starts so you only have to do it once. The file shouldn't be changing whilst this process is in operation.

File: appconfig/settings/iexfeed.q [hunk 2]
```diff
@@ -9,22 +9,16 @@ CONNECTIONS:enlist `tickerplant         // Feedhandler connects to the tickerpla
 HOPENTIMEOUT:30000
 
 \d .iex
-main_url:"https://api.iextrading.com"
-convert_epoch:{"p"$1970.01.01D+1000000j*x}
+mainUrl:"https://api.iextrading.com"
+convertEpoch:{"p"$1970.01.01D+1000000j*x}
 reqtype:`both
-syms:`CAT`DOG
+syms::`CAT`DOG
```
Comment (jemmaborland) [inline appconfig/settings/iexfeed.q]: is the extra `:` intentional here?

File: appconfig/settings/iexfeed.q [hunk 3]
```diff
@@ -9,22 +9,16 @@ CONNECTIONS:enlist `tickerplant         // Feedhandler connects to the tickerpla
...
 callback:".u.upd"
-data_suffix:{[sym]"/1.0/tops?symbols=",sym}
-ex_suffix:{[sym] "/1.0/stock/",sym,"/quote"}
+dataSuffix:{[sym]"/1.0/tops?symbols=",sym}
+exSuffix:{[sym] "/1.0/stock/",sym,"/quote"}
```
Comment (jemmaborland) [inline appconfig/settings/iexfeed.q]: Whitespace between `[sym] and "` - either keep it or don't (lines 17 and 18)

File: code/iexfeed/iex.q [hunk 4]
```diff
@@ -1,95 +1,96 @@
...
 exInfoPeriod:@[value;`exInfoperiod;0D01:00:00.000];
 lastTUD:lastQUD:syms!count[syms]#`float$();
 exInfo:([sym:`symbol$()]exName:`symbol$();openTime:`timestamp$();closeTime:`timestamp$());
 
 init:{[x]
-   if[`main_url in key x;.iex.main_url:x `main_url];
-   if[`quote_suffix in key x;.iex.quote_suffix:x `quote_suffix];
-   if[`trade_suffix in key x;.iex.trade_suffix:x`trade_suffix];
+   if[`mainUrl in key x;.iex.main_url:x `main_url];
+   if[`quoteSuffix in key x;.iex.quoteSuffix:x `quoteSuffix];
```
Comment (jemmaborland) [inline code/iexfeed/iex.q]: whitespace

File: code/iexfeed/iex.q (hunk 1 above)
Comment (jemmaborland) [reply code/iexfeed/iex.q]: All of the types should be in upper-case:

>Syntax: (types;delimiter ) 0: y
>
>Where y is a file descriptor, a string, or a list of strings, returns a vector or matrix interpreted from the content of y, where
>
>- types is a list of types in upper case,
>- delimiter is a char atom or 1-item list,

File: appconfig/settings/iexfeed.q [hunk 5]
```diff
@@ -9,22 +9,16 @@ CONNECTIONS:enlist `tickerplant         // Feedhandler connects to the tickerpla
 HOPENTIMEOUT:30000
 
 \d .iex
-main_url:"https://api.iextrading.com"
-convert_epoch:{"p"$1970.01.01D+1000000j*x}
+mainUrl:"https://api.iextrading.com"
+convertEpoch:{"p"$1970.01.01D+1000000j*x}
```
Comment (jonathonmcmurray) [inline appconfig/settings/iexfeed.q]: camelCase? ew

Comment (ThomasSmyth) [reply appconfig/settings/iexfeed.q]: camelCase? Beautiful

File: appconfig/settings/iexfeed.q [hunk 6]
```diff
@@ -9,22 +9,16 @@ CONNECTIONS:enlist `tickerplant         // Feedhandler connects to the tickerpla
...
 reqtype:`both
 syms:`CAT`DOG
 callback:".u.upd"
-data_suffix:{[sym]"/1.0/tops?symbols=",sym}
-ex_suffix:{[sym] "/1.0/stock/",sym,"/quote"}
-upd:{[t;x] .iex.callbackhandle(.iex.callback;t; value flip delete time from x)}
+dataSuffix:{[sym]"/1.0/tops?symbols=",sym}
+exSuffix:{[sym]"/1.0/stock/",sym,"/quote"}
```
Comment (claire-sloan) [inline appconfig/settings/iexfeed.q]: closing semi-colons

File: code/iexfeed/iex.q [hunk 7]
```diff
@@ -1,95 +1,97 @@
...
+   .iex.timer:$[not .iex.reqtype in key .iex.timerDict;'`timer;.iex.timerDict .iex.reqtype];
    };
 
-get_data:{[main_url;suffix].Q.hg`$main_url,suffix};
+getData:{[mainUrl;suffix].Q.hg`$mainUrl,suffix};
 
-format_data:{[data;cfg]
-   :update .iex.convert_epoch srctime,ex:" " from cfg[`cols] xcol enlist cfg[`types]$cfg[`scols]#data;
+formatData:{[data;cfg]
+   :update .iex.convertEpoch srctime from cfg[`cols] xcol enlist cfg[`types]$cfg[`scols]#data;
```
Comment (claire-sloan) [inline code/iexfeed/iex.q]: can make one liner

**Pull request merged**